web/ui/react-app/src/pages/graph/CMTheme.tsx

```diff
@@ -220,6 +238,27 @@ export const darkTheme = EditorView.theme(
         backgroundColor: '#767676',
       },
     },
+
+    '.cm-matchingBracket, &.cm-focused .cm-matchingBracket': {
+      color: '#1e293b',
+      backgroundColor: '#cbd5e1',
```

Comment:
Nice. But in dark mode, the cursor is now very low contrast in comparison to the bright background of the parentheses it is on, making it hard to spot. How about making that background color just a tad less bright, like the `#767676` one we have for selections?

Reply:
true, with #767676 the contrast ratio between the highlighted text and the background is a bit too low (not critical IMHO considering that this not really for reading, but more for highlighting a position within the query). We could also make the text color completely white or black to make it easier to read. What do you think?

Reply:
That's a good point. Looking at how it works for the light mode and playing with that, it would be nice to do the equivalent of that for dark mode:

* Don't override the text color at all (keep it at its default white-ish tone), and the boldness still gets added anyway.
* Set the background color to something subtle relative to the black background, like `#616161`.

![parens](https://user-images.githubusercontent.com/538008/181756300-9e30548a-d235-47b7-9608-b9f614e3a760.png)

So basically remove the `color` line above and do `backgroundColor: #616161`. What do you think?

Reply:
Indeed, removing the `color` and using the new background color provides a good enough contrast in this case.